src/files/index.js

```diff
@@ -31,6 +37,36 @@ class Files {
     this._sdk = sdk;
   }
 
+  /**
+   * Gets a temporary URL for the file in S3.
```

Comment:
Can we get rid of this reference to S3 here and anywhere else in this project? The underlying implementation details are not important

Reply:
Good call

Reply:
This is the only reference, removed it.